README.rst

```diff
@@ -4,6 +4,21 @@
   .. image:: https://coveralls.io/repos/broadinstitute/oncotator/badge.png?branch=issue_211_travis
     :target: https://coveralls.io/r/broadinstitute/oncotator?branch=issue_211_travis
 
+======================
+ONCOTATOR IS NO LONGER SUPPORTED OR MAINTAINED
+======================
+
+For Functional Annotations, please use Funcotator (a GATK tool):
```

Comment:
Maybe we should be a bit more verbose about this. 

Something like:
```
Funcotator is a re-engineered and improved version of Oncotator, it is:
  * Better: many bugs have been fixed and edge cases have been improved.
  * Faster: annotate more variants in less time
  * Easier to use and deploy: a single jar with no tricky installation or dependencies, and a tool for fetching the datasources:

Funcotator is available as part of the GATK toolkit and works out of the box with both Mutect2 and HaplotypeCaller for somatic and germline annotation.  
```

lets really try to sell it.

Reply:
I have a bulleted list of things Funcotator does that Oncotator doesn't do in the readme - I can duplicate that here in addition.

Also, Funcotator is _specifically *not*_ a new version of Oncotator.  It is a completely separate tool that was built from the ground up which does something very similar.

Reply:
Heh, sorry!  Touchy subject :)  Is it fair to say that from an oncotator user's perspective it can mostly be seen as an upgraded version?

Reply:
:P It's not actually a touchy subject - I just want to be absolutely clear to anyone who sees this because the names are similar (which is derived from what the software does, not its origins).

In some ways it's an upgrade, in others there are some limitations from what Oncotator provides (mostly in terms of supported data source file formats).  Like I said, the [Funcotator blog post](https://gatk.broadinstitute.org/hc/en-us/articles/360035889931-Funcotator-Information-and-Tutorial) has a full comparison with Oncotator.